From: Jaldhar H. Vyas <jaldhar@debian.org>
Date: Fri, 16 Sep 2011 00:55:02 +0200
Subject: default location for user mailboxes
Description: Set a default for the case where a user has no mail so auto-
 detection of mailboxes doesn't work.  mail_access_groups is also set to mail
 so that dovecot can access an mbox in the default location of /var/mail.
Bug: #623440, #824212
Last-Update: 2016-07-22
---
 doc/example-config/conf.d/10-mail.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/example-config/conf.d/10-mail.conf b/doc/example-config/conf.d/10-mail.conf
index ea4c9e0..90a57e0 100644
--- a/doc/example-config/conf.d/10-mail.conf
+++ b/doc/example-config/conf.d/10-mail.conf
@@ -27,7 +27,7 @@
 #
 # <doc/wiki/MailLocation.txt>
 #
-#mail_location = 
+mail_location = mbox:~/mail:INBOX=/var/mail/%u
 
 # If you need to set multiple mailbox locations or want to change default
 # namespace settings, you can do it by defining namespace sections.
@@ -118,7 +118,7 @@ namespace inbox {
 # dangerous to set these if users can create symlinks (e.g. if "mail" group is
 # set here, ln -s /var/mail ~/mail/var could allow a user to delete others'
 # mailboxes, or ln -s /secret/shared/box ~/mail/mybox would allow reading it).
-#mail_access_groups =
+mail_access_groups = mail
 
 # Allow full filesystem access to clients. There's no access checks other than
 # what the operating system does for the active UID/GID. It works with both
